element_interface/utils.py

```diff
@@ -140,31 +147,55 @@ def ingest_csv_to_table(
         )
         if verbose:
             insert_len = len(table) - prev_len
-            print(
+            logger.info(
                 f"\n---- Inserting {insert_len} entry(s) "
-                + f"into {table.table_name} ----"
+                + f"into {to_camel_case(table.table_name)} ----"
             )
 
 
-def recursive_search(key: str, dictionary: dict) -> any:
-    """Return value for key in a nested dictionary
-
-    Search through a nested dictionary for a key and returns its value.  If there are
-    more than one key with the same name at different depths, the algorithm returns the
-    value of the least nested key.
+def value_to_bool(value) -> bool:
+    """Return whether the provided value represents true. Otherwise false.
 
     Args:
-        key (str): Key used to search through a nested dictionary
-        dictionary (dict): Nested dictionary
+        value (str, bool, int): Any input
 
     Returns:
-        value (any): value of the input argument `key`
+        bool (bool): True if value in ("y", "yes", "t", "true", "on", "1")
     """
-    if key in dictionary:
-        return dictionary[key]
-    for value in dictionary.values():
-        if isinstance(value, dict):
-            a = recursive_search(key, value)
-            if a is not None:
-                return a
-    return None
+    if not value:
+        return False
+    return str(value).lower() in ("y", "yes", "t", "true", "on", "1")
+
+
+def write_csv(content: list, path: pathlib.PosixPath):
```

Comment:
Why does this function need to exist?  Its body can be used directly inline with greater clarity.

Reply:
You previously critiqued element workflow pytests for needlessly relying on pandas ([discussion](https://github.com/datajoint/workflow-session/pull/9#discussion_r775087876)). To remove the pandas dependency, we added `write_csv` to a number of workflows ([ephys](https://github.com/datajoint/workflow-array-ephys/blob/main/tests/__init__.py#L38), [dlc](https://github.com/datajoint/workflow-deeplabcut/blob/main/tests/conftest.py#L91), [miniscope](https://github.com/datajoint/workflow-miniscope/blob/main/tests/__init__.py#L30), [session](https://github.com/datajoint/workflow-session/blob/main/tests/__init__.py#L29)). I found more clarity removing this body from each of the various pytest fixtures and, in this PR, I centralize. Could we rename it? `write_list_to_file`?

Reply:
Writing lines to a text file already exists in pathlib. You really don't need a function.
```python

Reply:
I was unaware. I'll plan to make this adjustment on the next update of each workflow

Reply:
Does CSV want an `\n` on the last line ideally?
You may want to do append one:
```python
path.write_text('\n'.join(lines) + '\n')
```

Reply:
I don't think the terminal `\n` is needed, no

Reply:
I think generally, all non-empty text files should end with a newline: https://thoughtbot.com/blog/no-newline-at-end-of-file. 
According to POSIX all lines, including the last one should be terminated by a newline. https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

I would think that `write_text` would add it automatically but I haven't checked.